docs/getting_started/command_line_interface.md

```diff
@@ -32,7 +32,7 @@ Detailed guides to each subcommand can be found [here](../tools.md).
 ### Script source
 
 Deno can grab the scripts from multiple sources, a filename, a url, and '-' to
-read the file from stdin. The later is useful for integration with other
```

Comment:
It's a typo - should be "latter".

Reply:
Ha, I first used latter, but I think that only works for two items, e.g. _former_ and _latter_? I decided to go for _last_ in the end, but I could change it back to _latter_ if that's preferred? :-)